Payment-Integration-Api.swagger.yml

```diff
@@ -1,108 +0,0 @@
-swagger: '2.0'
```

Comment:
I liked the idea of having swagger API spec. Of course it was not maintained but maybe we should address it at some point?

Reply:
Yes, I liked it too but the one here was very different to the one actually implemented so I think it's easier to add a new one instead of publishing a wrong one.